lib/crypto/butil.js

```diff
@@ -35,7 +35,7 @@ module.exports.xorBuffers = function xorBuffers(buffer1, buffer2) {
 module.exports.unbuffer = function unbuffer(object, inplace) {
   var keys = Object.keys(object)
   var copy = inplace ? object : {}
-  for (var i = 0; i < keys.length; i++) {
+  for (var i = 0; i < keys.length; i ++) {
```

Comment:
Using @pdehaan's rule (modified to add quotes around error), the content server passes!

```
'space-unary-ops': ['error', {words: true, nonwords: false, overrides: {'!': true, '!!': true}}],
```

```bash
➜  fxa-content-server git:(master) grunt lint                                       git:(master|)
Running "eslint:app" (eslint) task

Running "eslint:grunt" (eslint) task

Running "eslint:scripts" (eslint) task

Running "eslint:server" (eslint) task

Running "eslint:tests" (eslint) task

Running "jsonlint:app" (jsonlint) task
>> 3 files lint free.

Running "amdcheck:tests" (amdcheck) task

Total unused paths: 0 in 0 files.
Total unused dependencies: 0 in 0 files.
Total processed files: 73

Running "sasslint:sass" (sasslint) task

Done.


Execution Time (2017-02-06 14:45:06 UTC-0)
loading tasks    2.5s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 20%
eslint:app       6.5s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 54%
eslint:grunt    374ms  ▇▇▇ 3%
eslint:server   391ms  ▇▇▇ 3%
eslint:tests     1.3s  ▇▇▇▇▇▇▇▇ 11%
jsonlint:app    335ms  ▇▇ 3%
amdcheck:tests  262ms  ▇▇ 2%
sasslint:sass   441ms  ▇▇▇ 4%
Total 12.1s

➜  fxa-content-server git:(master)                                                  git:(master|)
```

However, with the `no-nested-ternary`, we do not. :(

```
'no-nested-ternary': 'error',
```

```bash
➜  fxa-content-server git:(master) grunt lint                                       git:(master|)
Running "eslint:app" (eslint) task

Running "eslint:grunt" (eslint) task

Running "eslint:scripts" (eslint) task

Running "eslint:server" (eslint) task

Running "eslint:tests" (eslint) task

/Users/shanetomlinson/development/fxa-content-server/tests/functional/fx_fennec_v1_force_auth.js
  30:29  error  Do not nest ternary expressions  no-nested-ternary

/Users/shanetomlinson/development/fxa-content-server/tests/functional/fx_ios_v1_sign_in.js
  46:29  error  Do not nest ternary expressions  no-nested-ternary

/Users/shanetomlinson/development/fxa-content-server/tests/functional/sync_force_auth.js
  33:29  error  Do not nest ternary expressions  no-nested-ternary

/Users/shanetomlinson/development/fxa-content-server/tests/functional/sync_sign_in.js
  42:29  error  Do not nest ternary expressions  no-nested-ternary

/Users/shanetomlinson/development/fxa-content-server/tests/functional/sync_v2_force_auth.js
  41:29  error  Do not nest ternary expressions  no-nested-ternary

/Users/shanetomlinson/development/fxa-content-server/tests/functional/sync_v2_sign_in.js
  37:29  error  Do not nest ternary expressions  no-nested-ternary

/Users/shanetomlinson/development/fxa-content-server/tests/functional/sync_v3_sign_in.js
  41:29  error  Do not nest ternary expressions  no-nested-ternary

✖ 7 problems (7 errors, 0 warnings)

Warning: Task "eslint:tests" failed. Use --force to continue.

Aborted due to warnings.


Execution Time (2017-02-06 14:46:47 UTC-0)
loading tasks   2.4s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 22%
eslint:app      6.4s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 59%
eslint:grunt   376ms  ▇▇▇ 3%
eslint:server  402ms  ▇▇▇ 4%
eslint:tests    1.3s  ▇▇▇▇▇▇▇▇ 12%
Total 10.9s

➜  fxa-content-server git:(master)
```